docs/samples/python/Presidio_with_Flair_as_a_3rd_party_recognizer

```diff
@@ -49,54 +75,74 @@ class FlairRecognizer(EntityRecognizer):
 
     # Class to use Flair with Presidio as an external recognizer.
     def analyze(
-        self, text: str, entities: List[str] = [], nlp_artifacts: NlpArtifacts = None
-    ) -> List[RecognizerResult]:
+        self, text: str, entities: List[str] = [], language: str = 'en'
```

Comment:
The language parameter is not passed by the `AnalyzerEngine`, so the user wouldn't be able to change this (as they are not interacting with the recognizer directly). It is better to load the specific language model on `__init__` and then apply the loaded model during the call to `analyze`

In addition, the call `self.model = SequenceTagger.load(self.MODEL_LANGUAGES.get(language))` would load the NLP model from disk on every request, which is not optimal.

Reply:
the `analyze` method must have the same signature as the parent class: https://github.com/microsoft/presidio/blob/4b0baec8f30fdd0bc85d9762ccb39176b2aeb305/presidio-analyzer/presidio_analyzer/entity_recognizer.py#L68

Otherwise we'll get a exception `TypeError: analyze() got an unexpected keyword argument 'nlp_artifacts'`

Reply:
I eliminate the language parameter in the analyze, and I put the loader in the __init__.